js/companies.js

```diff
@@ -21,36 +25,68 @@ var Companies = ( () => {
             return companyContainer[name];
         },
 
+        /* addByPages is used by tab.js to count only unique 
+         * tracking networks on a tab
+         */
+        addByPages: (name) => {
+            if(!companyContainer[name]){
+                companyContainer[name] = new Company(name);
+                topBlocked.add(name);
+            }
+            if (name !== 'unknown') companyContainer[name].incrementPagesSeenOn();
+        },
```

Comment:
`addByPages()` does not describe what's happening here. Maybe `countCompanyOnPage()` ?

Reply:
👍